packages/dd-trace/test/appsec/iast/analyzers/insecure-cookie-analyzer.express.plugin.spec.js

```diff
@@ -0,0 +1,72 @@
+'use strict'
+
+const { prepareTestServerForIastInExpress } = require('../utils')
+const fs = require('fs')
+const os = require('os')
+const path = require('path')
+describe('Insecure cookie vulnerability', () => {
+  let setCookieFunctions
+  const setCookieFunctionsFilename = 'set-cookie-express-functions.js'
+  const setCookieFunctionsPath = path.join(os.tmpdir(), setCookieFunctionsFilename)
+
+  before(() => {
+    fs.copyFileSync(path.join(__dirname, 'resources', 'set-cookie-express-functions.js'), setCookieFunctionsPath)
+    setCookieFunctions = require(setCookieFunctionsPath)
+  })
+
+  after(() => {
+    fs.rmSync(setCookieFunctionsPath)
+  })
+
+  withVersions('express', 'express', version => {
+    prepareTestServerForIastInExpress('in express', version,
+      (testThatRequestHasVulnerability, testThatRequestHasNoVulnerability) => {
+        testThatRequestHasVulnerability((req, res) => {
+          setCookieFunctions.insecureWithResCookieMethod('insecure', 'cookie', res)
+        }, 'INSECURE_COOKIE', {
```

Comment:
We can use the value from [packages/dd-trace/src/appsec/iast/vulnerabilities.js](https://github.com/DataDog/dd-trace-js/pull/3184/files#diff-359177f30c6a634464ec2d384a56ffb26eaacd102dabed0ba430ff1e7571d0d9R8)